overlord/snapstate/snapmgr.go

```diff
@@ -99,6 +101,10 @@ func (ss *SnapSetup) TryMode() bool {
 	return ss.Flags&TryMode != 0
 }
 
+func (ss *SnapSetup) RevertOp() bool {
```

Comment:
This seems slightly off from what we do on other cases. We don't have, e.g. InstallOp, RemoveOp, etc, so what's different about revert that makes it necessary?  I'd expect revert to have its revision set on Revision itself, and the tasks to do the right thing.